_translator-files/po/zh_CN/Installation-for-Windows.po

```diff
@@ -1,289 +1,290 @@
-msgstr ""
+msgstr "**下载并安装 ASIO 驱动程序**。尝试使用硬件制造商提供的驱动程序。如果你找不到，或者你没有外置声卡，你可能需要安装 ASIO4ALL。有关更多信息，请向下滚动到 [ASIO](#asio) 部分。\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:20
 #, no-wrap
 msgid "[Download Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
-msgstr ""
+msgstr "[下载 Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:21
 #, no-wrap
 msgid "**Mirror 2:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
-msgstr ""
+msgstr "**镜像源 2:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:22
 #, no-wrap
 msgid "**Install Jamulus**: Double click the installer to launch it. If you get a warning notice from SmartScreen, click on \"More info\" and \"Run anyway\" to install Jamulus. (If you grabbed a new version of Jamulus and are one of the first people who downloaded it, Jamulus won't be whitelisted by SmartScreen yet. We don't pay for code signing.)\n"
-msgstr ""
+msgstr "**安装 Jamulus**:双击安装程序以启动它。如果您从 SmartScreen 收到警告通知，请单击“更多信息”和“仍然运行”以安装 Jamulus。 （如果您获得了 Jamulus 的新版本并且是第一批下载它的人之一，那么 Jamulus 不会被 SmartScreen 列入白名单。我们不为代码签名付费。）\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:23
 #, no-wrap
 msgid "**Run Jamulus**. Now you should be able to use Jamulus just like any other application.\n"
-msgstr ""
+msgstr "**运行 Jamulus**。现在您应该能够像使用任何其他应用程序一样使用 Jamulus。\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:24
 #, no-wrap
 msgid "**Set up your sound card**. When you're done, you need to set up your audio hardware. And if you are using ASIO4ALL, have a look at how to set that up below.\n"
-msgstr ""
+msgstr "**设置您的声卡**。完成后，您需要设置音频硬件。如果您使用的是 ASIO4ALL，请查看下面的设置方法。\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:26
 msgid "_Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._"
-msgstr ""
+msgstr "_请注意，在未获得 [Steinberg 的许可协议](https://www.steinberg.net/developers/) 的情况下，您不得重新分发此二进制文件。_"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:28
 msgid "If you don't want to use ASIO or prefer [JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html), you can also use the [JACK version of Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }})."
-msgstr ""
+msgstr "如果您不想使用 ASIO 或者更喜欢 [JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html)，也可以使用 [JACK 版本的 Jamulus]({{ site.download_root_link } {{ site.download_file_names.windows-jack }}）。"
```

Comment:
There's a closing curly bracket missing in `{{ site.download_root_link }`.

Reply:
Fix For now